components/backstage/backstage_controller.jsx

```diff
@@ -63,6 +63,7 @@ export default class BackstageController extends React.Component {
 
         siteName: PropTypes.string,
         enableCustomEmoji: PropTypes.bool.isRequired,
+        canManageIntegrations: PropTypes.bool.isRequired,
```

Comment:
Super minor nit: can we move this down next to the `canCreateOrDeleteCustomEmoji` so it doesn't break the flow here of all the enable props together?